escher/shaders/lighting/occlusion_detector.h

```diff
@@ -28,7 +28,7 @@ class OcclusionDetector {
   GLint position() const { return position_; }
 
   // Must match fragment shader.
-  static const int kNoiseSize = 5;
+  static const int kNoiseSize = 8;
```

Comment:
This also needs to match the radius in the IlluminationReconstructionFilter.  I have a different fix in https://github.com/domokit/waterfall/pull/25 that is probably a bit better.  Let's land your patch and I'll rebase on top of it.